Sources/PointFree/Blog/BlogPostShow.swift

```diff
@@ -8,7 +8,7 @@ import Prelude
 import Styleguide
 import Tuple
 
-let blogPostShowMiddleware: (Conn<StatusLineOpen, Tuple4<Either<String, Int>, Database.User?, SubscriberState, Route?>>) -> IO<Conn<ResponseEnded, Data>> =
```

Comment:
I got a small boost in compilation by moving the types down to `layoutData` instead of here, so just playing with that.

Reply:
huh